.gitlab/pipelines/runtime/config.sh

```diff
@@ -2,8 +2,9 @@
 
 export KUBECF_NAMESPACE="kubecf"
 export CF_OPERATOR_NAMESPACE="cfo"
-export CLUSTER_IP="10.43.0.255"
-export SYSTEM_DOMAIN="${CLUSTER_IP}.nip.io"
+export ROUTER_CLUSTER_IP="10.43.0.250"
+export SSH_PROXY_CLUSTER_IP="10.43.0.251"
+export SYSTEM_DOMAIN="kubecf-ci.susecap.net"
```

Comment:
Not a fan of using the internal-development DNS here.  We should _at least_ be using `kubecf.suse.dev` or something instead.

Not to mention, `kubecf-ci` isn't well namespaced… that domain is _already_ full of hard-to-read-through crap.

Reply:
Updated